rawsnippets/Pipeline-with-multiple-task.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
This pipeline template should be extended, you just referring to commands but template doesn't provide configuration for them.